master/master.cfg

```diff
@@ -340,7 +340,7 @@ class BuilderType:
         return (self.arch == 'x86'
                 and self.bits == 64
                 and self.os == 'linux'
-                and self.halide_branch == HALIDE_MAIN)
+                and self.halide_branch not in [HALIDE_RELEASE_14, HALIDE_RELEASE_15]
```

Comment:
I think it would be better to specify the ones we want, not the ones we don't want: `in [HALIDE_MAIN, HALIDE_RELEASE_16]`